content/courses/advanced-ada/chapters/limited_types.rst

```diff
@@ -897,47 +897,159 @@ of the task's entries, because that would deadlock. That is, the entry
 call would wait until the task reaches an accept statement, which will
 never happen, because the task will never be activated.
 
-Other usages of extended return statements
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Initialization and function return
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As mentioned in the previous section, the object of limited type returned by
+the initialization function is created *in place*. In other words, the return
+object is created at the location where the initialization function is being
+called.
```

Comment:
"created at the location where the initialization function is being called." isn't quite right. It is built in the target of the assignment, ie the left-hand side. So maybe "built in the object that is the target of the assignment statement."

Reply:
Yes, I agree the original phrase is misleading. I've just replaced it with your suggestion.